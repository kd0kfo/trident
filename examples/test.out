
   \ ,-*-.   ,-*-. ,-*-.   ,-*-. ,-*-.   ,    
    X | | \ / | | X | | \ / | | X | | \ /   
   / \| | |X| | |/ \| | |X| | |/ \| | |X|   
      `-!-' `-!-'   `-!-' `-!-'   `-!-' `-     

   Trident 0.4.8 microRNA dsDNA triplex algorithm

                ,-*-.         ,-*-.            
               / | | \       / | | \         
         | | |/       \| | |/       \| |     
         `-!-'         `-!-'         `-!       


License: GPLv3
Query Filename: 	hsa-miR-483-5p.fasta
Reference Filename:	dna_segment.fasta
Read Sequence:hsa-miR-483-5p MIMAT0004761 Homo sapiens miR-483-5p(22 nt)
Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

   Forward:	Score: 260.000000  Q:2 to 22  R:12 to 33 Align Len (21) (76.19%) (76.19%)

   Query:    3' GAGGGAAGAAAGGAGGGCAGAa 5'
                $ $$$$$$$$  $ $$$ $$$ 
   Ref:      5' GGGGGAAGAAGAGGGGGCAGAg 3'
             3' CCCCCTTCTTCTCCCCCGTCTc 5'

   Energy:  -327.920000 kCal/Mol

Scores for this hit:
>hsa-miR-483-5p,DNA,260.00,-327.92,2 22,12 33,21,76.19%,76.19%,GAGGGAAGAAAGGAGGGCAGAa,$ $$$$$$$$  $ $$$ $$$ ,GGGGGAAGAAGAGGGGGCAGAg,antiparallel,direct,purine
Score for this Scan:
Seq1,Seq2,Tot Score,Tot Energy,Max Score,Max Energy,Strand,Len1,Len2,Positions
>>hsa-miR-483-5p	DNA	260.00	-327.92	260.00	-327.92	1	22	45	 12
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

   Forward:	Score: 176.000000  Q:6 to 22  R:9 to 31 Align Len (18) (72.22%) (72.22%)

   Query:    3' GAGGGAAGA-AAGGAGGGcagaa 5'
                $$ $$  $$ $$$ $$$$     
   Ref:      3' GACGGGGGAGAAGAAGGGggtca 5'
             5' CTGCCCCCTCTTCTTCCCccagt 3'

   Energy:  -264.290000 kCal/Mol

Scores for this hit:
>hsa-miR-483-5p,DNA,176.00,-264.29,6 22,9 31,18,72.22%,72.22%,GAGGGAAGA-AAGGAGGGcagaa,$$ $$  $$ $$$ $$$$     ,GACGGGGGAGAAGAAGGGggtca,parallel,direct,purine
Score for this Scan:
Seq1,Seq2,Tot Score,Tot Energy,Max Score,Max Energy,Strand,Len1,Len2,Positions
>>hsa-miR-483-5p	DNA	176.00	-264.29	176.00	-264.29	2	22	45	 15
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Score for this Scan:
No Hits Found above Threshold
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Score for this Scan:
No Hits Found above Threshold
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Score for this Scan:
No Hits Found above Threshold
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Score for this Scan:
No Hits Found above Threshold
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Score for this Scan:
No Hits Found above Threshold
Complete

Read Sequence:DNA Segment from KDM6B(45 nt)
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Performing Scan: hsa-miR-483-5p vs DNA
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Score for this Scan:
No Hits Found above Threshold
Complete

Scan Complete

