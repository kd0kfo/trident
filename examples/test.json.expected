{
"run_info": {"version": "0.4.8", "build_time": "Fri Oct 25 10:24:12 CDT 2013", "build_sha": "70fe6cc55dd9e77eeff33f9cd907a7e4fe02eb20", "run_time": "Fri Oct 25 10:24:33 2013"},
"data": [
{
	"query_id": "hsa-miR-483-5p",
	"reference_id": "DNA",
	"score": 260.00,
	"energy": -327.92,
	"query_start": 2,
	"query_end": 22,
	"ref_start": 12,
	"ref_end": 12,
	"alignment_length": 21,
	"identity": 76.19,
	"similarity": 76.19
}
,
{
	"query_id": "hsa-miR-483-5p",
	"reference_id": "DNA",
	"score": 176.00,
	"energy": -264.29,
	"query_start": 6,
	"query_end": 22,
	"ref_start": 9,
	"ref_end": 9,
	"alignment_length": 18,
	"identity": 72.22,
	"similarity": 72.22
}
] }
